src/ImageSharp.Drawing/Processing/Processors/Drawing/DrawImageProcessor.cs

```diff
@@ -81,6 +81,12 @@ protected override void OnFrameApply(ImageFrame<TPixelDst> source, Rectangle sou
 
             var workingRect = Rectangle.FromLTRB(minX, minY, maxX, maxY);
 
+            if (workingRect.Width < 0 || workingRect.Height < 0)

```

Comment:
Wondering if this should be `<= 0` instead?

Reply:
It should be yes. Changed in 4411ddd